packages/epics/src/membership/components/space-group-slider/SpaceGroupSlider.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { Text } from '@radix-ui/themes';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  Button,
+} from '@hypha-platform/ui';
+import Link from 'next/link';
+import { SpaceType } from '@hypha-platform/graphql/rsc';
+import { Locale } from '@hypha-platform/i18n';
+import { getDhoPathAgreements } from '../../../../../../apps/web/src/app/[lang]/dho/[id]/constants';
```

Comment:
Please never import app components from packages. You're creating circular imports. A package like epics should never relate to implementations within apps. Either pass in the `href` or the getter function.